WebRoot/WEB-INF/tags/infra/view.tag

```diff
@@ -473,7 +473,8 @@ if (screen.width<=800) {
 											<a class='skin_link' target="_new" href="${adminReference}"><fmt:message key="adminLinkLabel"/></a>&nbsp;<font color="gray">|</font>&nbsp;
 										</c:if>
 											<!--<a class='skin_yahoo_link' target="_new" href="<fmt:message key='yahooYahooMailURL'/>"><fmt:message key='yahooYahooMail'/></a>&nbsp;<font color="gray">|</font>&nbsp;-->
-											<a class='skin_link' target="_new" href="<c:url value="${helpUrl}"><c:param name='locid'><fmt:getLocale /></c:param></c:url>"><fmt:message key="help"/></a> &nbsp; 
```

Comment:
Can you just remove this line instead of commenting it out.

Reply:
Yes I can remove this link but I have commented this for future reference.

Reply:
For future reference we should be able to look at history of file

Reply:
Removed.